config/settings/base.py

```diff
@@ -35,6 +35,7 @@
 # ------------------------------------------------------------------------------
 # Hosts/domain names that are valid for this site
 ALLOWED_HOSTS = env.list('DJANGO_ALLOWED_HOSTS', default=['*'])
+USE_X_FORWARDED_HOST = env.bool('DJANGO_USE_X_FORWARDED_HOST', default=False)
```

Comment:
For `env.bool` and `env.str`, we don't actually need the `default=` kwarg, just using an arg is enough. For `env.list` this is needed, unless `cast` is also defined as an arg.

Some existing values in the SODAR config have the unnecessary `default` kwargs, I believe these came from the original Cookiecutter template. Those will be cleaned up eventually as well :)

Reply:
I'm afraid we actually need "default=False" in the boolean case.
If I remove it, building the Docker image will fail like this:

`django.core.exceptions.ImproperlyConfigured: Set the DJANGO_USE_X_FORWARDED_HOST environment variable`